<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditerClasses</class>
 <widget class="QDialog" name="EditerClasses">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="lay_haut">
     <item>
      <widget class="QPushButton" name="supprime_classe">
       <property name="text">
        <string>Supprimer la classe actuelle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="select_classe"/>
     </item>
     <item>
      <widget class="QPushButton" name="ajoute_classe">
       <property name="text">
        <string>Ajouter une nouvelle classe</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lay_bas">
     <item>
      <layout class="QVBoxLayout" name="lay_gauche">
       <item>
        <widget class="QPushButton" name="ajoute_sort">
         <property name="text">
          <string>Ajoute un sort</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tab_sorts">
         <column>
          <property name="text">
           <string>Sorts</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="lay_droite">
       <item>
        <layout class="QHBoxLayout" name="lay_propox">
         <item>
          <widget class="QLabel" name="text_propx">
           <property name="text">
            <string>proportion x</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="propx"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="lay_propy">
         <item>
          <widget class="QLabel" name="text_propoy">
           <property name="text">
            <string>proportion y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="propy"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="lay_arme">
         <item>
          <widget class="QLabel" name="text_arme">
           <property name="text">
            <string>Arme principale</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="arme"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="lay_metier">
         <item>
          <widget class="QLabel" name="text_metier">
           <property name="text">
            <string>MÃ©tier principal</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="metier"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditerClasses</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditerClasses</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
